test/es6/module-syntax.js

```diff
@@ -124,6 +124,16 @@ var tests = [
             assert.doesNotThrow(function () { WScript.LoadModuleFile('.\\module\\ValidReExportStatements.js', 'samethread'); }, "Valid re-export statements");
         }
     },
+    {
+        name: "HTML comments do not parse",
```

Comment:
nit: "... do not parse in module scripts"

Just to be clear about the intent and scope of this test, that yes, indeed, this is specific to modules.